### PR TITLE
Design System: Set border radius to theme.shape.radius.default

### LINKED ISSUE #65089: Rework: Set border radius to theme.shape.radius.default
@tskarhed added a [default token for border radius](https://github.com/grafana/grafana/pull/64502) which needs to be added to the issues that were already completed. The list below contains all items that need to be reworked.

- [x] public/app/features/logs/components/getLogRowStyles.ts
- [ ] public/app/plugins/datasource/graphite/components/MetricTankMetaInspector.tsx
- [x] public/app/features/explore/ExploreDrawer.tsx
- [ ] public/app/features/explore/Graph/ExploreGraph.tsx
- [x] public/app/features/explore/NoData.tsx
- [x] public/app/features/explore/TraceView/components/TracePageHeader/TracePageSearchBar.tsx
- [ ] public/app/features/explore/TraceView/components/TraceTimelineViewer/SpanBar.tsx
- [ ] public/app/features/explore/TraceView/components/TraceTimelineViewer/SpanBarRow.tsx
- [x] public/app/features/explore/TraceView/TraceViewContainer.tsx

### PR DESCRIPTION
**Pull request by L-M-K-B**
Fixes #65089

### CODE REVIEW

**Review by adrapereira — APPROVED** [review]

**Review by matyax — APPROVED** [review]
Logs part looks good 👍

File: public/app/features/explore/ExploreDrawer.tsx [hunk 1]
```diff
@@ -35,15 +35,14 @@ const getStyles = stylesFactory((theme: GrafanaTheme2) => {
       animation: 0.5s ease-out ${drawerSlide(theme)};
     `,
     rzHandle: css`
-      background: ${theme.colors.secondary.main};
```
Comment (ivanahuckova) [inline public/app/features/explore/ExploreDrawer.tsx:38]: Is this removed on purpose?

Comment (ivanahuckova) [reply public/app/features/explore/ExploreDrawer.tsx:38]: In UI, the handle is not visible now. 
Now:
image

Before: 
image

**Review by ivanahuckova — CHANGES_REQUESTED** [review]

File: public/app/features/explore/ExploreDrawer.tsx (hunk 1 above)
Comment (L-M-K-B) [reply public/app/features/explore/ExploreDrawer.tsx]: No, I removed it by mistake. Now it looks fine again.

**Review by ivanahuckova — APPROVED** [review]

File: public/app/features/explore/ExploreDrawer.tsx [hunk 2]
```diff
@@ -43,7 +43,7 @@ const getStyles = stylesFactory((theme: GrafanaTheme2) => {
       left: calc(50% - 100px) !important;
       top: -4px !important;
       cursor: grab;
-      border-radius: 4px;
+      border-radius: ${theme.shape.borderRadius(2)};
```
Comment (JoaoSilvaGrafana) [inline public/app/features/explore/ExploreDrawer.tsx:46]: In this case we can use the `theme.shape.radius.pill` 😄

Comment (L-M-K-B) [reply public/app/features/explore/ExploreDrawer.tsx]: Just checked whether `theme.shape.radius.pill` changes it drastically and it doesn't. So I changed it.
Before:
Screenshot 2023-03-29 at 14 50 33
After:
Screenshot 2023-03-29 at 14 57 38

**Review by JoaoSilvaGrafana — APPROVED** [review]
lgtm! 🎉

**Review by gelicia — APPROVED** [review]
ExploreDrawer handle looks fine and the CSS shows `9999px` for the border radius as expected.

**Pull request merged**